### ISSUE TITLE
About training code

### ISSUE THREAD

**Issue by d12306** [bethgelab/AnalysisBySynthesis#4]
@lukas-schott , thank you for your implementation, can you also show the training code in order to me to evaluate it on other datasets?

Thanks

**Comment by lukas-schott** [comment]
As our training code is intertwined with other projects, we do not plan on releasing it.

However, it should be straight forward to train one VAE for each class. All relevant hyperparameters should be in the paper. Otherwise, I am happy to answer any questions regarding the training procedure.

**Comment by d12306** [comment]
Thanks, I will try to train it and see what happens.

**Comment by Jindong-Explainable-AI** [comment]
@lukas-schott  Thanks for your awesome project and code! I am trying to train VAEs using the code. The loss function I used is ELBOs() in the abs/abs_models/loss_functions.py. However, the trained models failed to generate realistic digits.  The loss function ELBOs() return elbo / (n_ch * nx * ny), which is not correct, in my opinion. There is no reason to divide elbo by the input dimensions. I appreciate your comments.

**Comment by lukas-schott** [comment]
dividing the ELBO by the input dimension in 
https://github.com/bethgelab/AnalysisBySynthesis/blob/master/abs_models/loss_functions.py#L37
is equivalent to adjusting the learning rate and can be seen as a hyperparameter.
